tests/acceptance/features/webUILogin/login.feature

```diff
@@ -70,3 +70,16 @@ Feature: login users
     Then the user should be redirected to the login error page
     When the user exits the login error page
     Then the user should be redirected to the login page
+
+  Scenario: the user session of a deleted user should not be valid for newly created user of same name
+    Given these users have been created with default attributes:
+      | username |
+      | user1    |
+    And user "user1" has logged in using the webUI
+    And the user has browsed to the files page
+    And user "user1" has been deleted
+    And the user has reloaded the current page of the webUI
+    And the user has been redirected to the login error page
```

Comment:
```suggestion
    When the user reloads the current page of the webUI
    Then the user should have been redirected to the login error page
```

Maybe this scenario can end here, and then have another scenario with similar given steps plus 
```
And user "user1" has been created with default attributes
```
which checks the case where the original user has been deleted, and the same user name has been created again, all while the original user stayed logged-in at the browser.

Because actually we probably want to cover both scenarios.

Reply:
Hmmm - that looks just like the scenario at line 62. So we already have done that.

So for this new scenario we can just move `And user "user1" has been created with default attributes` up to be straight after `And user "user1" has been deleted` and then tthe user can reload the page and test what happens